index.rst

```diff
@@ -127,214 +138,45 @@ maintained by Confluent Inc., the team that created Kafka.
-This filtered stream can then be read by another consumer or directed
-to another output method for storage.
-The downstream consumer of the filtered stream can itself be another
-filter, meaning that this scheme allows simple chaining of filters.
-
-Currently, filters are defined in ``lsst.alert.stream.filters`` in the
-alert_stream repository.
-Each filter is constructed as a class with a filter function.
-Below is an example of code defining a simple filter.
-
-.. code-block:: Python
-
-  class Filter001(AlertFilter):
-      def filter(self, alert):
-          if ((alert['diaSource']['snr'] > 5) &
-              (alert['diaSource']['diffFlux'] > 0.00003631)):  # 20th mag
-              return True
-          else:
-              return False
-
-Filters inherit from an AlertFilter base class.
-When the filter class is called on each alert, the visit ID is read.
-Up to 20 alerts per visit that pass the filter, i.e., return True, are
-forwarded to its local Kafka instance to a topic for the filtered alerts,
-named for the class.
-In the above example, alerts are sent to a new topic named "Filter001."
-The filter detects the beginning of a new visit when the visit ID
-changes from one alert to the next.
-This assumes that all alerts from a visit will be received before
-the next visit's alerts arrive.
-Otherwise, more than 20 alerts may pass through the filter.
-
-An excess of 20 alerts is not cause for concern but is defined by
-``numBrokerAlerts`` (Number of full-sized alerts that can be received
-per visit per user) in performance requirement DMS-REQ-0343 which
-was put in place as a coarse load-balancing measure.
-Raising this limit would potentially produce scientific gains as a larger
-number of alerts of interest would be available to scientific users.
-Currently, the first 20 alerts passing the filter are made available
-to users, and there is no ability to rank order or otherwise choose
-the most scientifically interesting 20 filtered alerts.
-Increasing the limit, however, would also increase the network
-bandwidth necessary to accommodate larger filtered streams.
-The current requirement for ``numBrokerUsers`` (Supported number of simultaneous
-users connected to the LSST alert filtering system) is set at 100,
-meaning that the system is at least required to support the
-bandwidth needed for 100 filtered streams with 20 alerts per visit.
-Increasing the filtered alert limit could then mean that fewer
-users would be able to be supported due to networking limitations.
-DMS-REQ-0343 notes that the requirement could also be satisfied
-by cutting down the content of filtered alerts and potentially
-removing fields to decrease the size of filtered streams.
+Selected community alert brokers will receive the full LSST alert stream and provide a range of user tools to identify alerts of interest.
+We are currently evaluating technical approaches for on-project filtering of the alert stream for users with LSST Data Rights (see :cite:`RDO-013`).
```

Comment:
What does "on-project" mean in this sentence?

Reply:
I'll revise to say "LSST-hosted."

Reply:
Okay. I think DMTN-165 goes even further. "Alerts on your laptop" means *filtering* of the alert stream happens off of LSST-hosted computers.

I don't really think it's worth our time to back-and-forth this one too much though - LSST-hosted is fine too.

Reply:
Yes, but DMTN-165 is definitely a scope change--so unless/until that is approved by project-level CCB I don't want to lean too hard into it.